modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3421,6 +3421,91 @@ definition {
 			actionLabel = "Update Action Label");
 	}
 
+	@description = "LPS-173537 - Verify adding a Commerce Product entry triggers an action to update the Commerce Product Group entry."
+	@priority = 5
+	test CanUpdateCommerceProductGroupEntryAfterCreatingCommerceProductEntry {
+		property portal.acceptance = "true";
+
+		task ("Given: A Commerce Product Group") {
+			CommerceNavigator.gotoPortlet(
+				category = "Pricing",
+				portlet = "Product Groups");
+
+			Click(locator1 = "Button#PLUS");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "IFrame#MODAL_IFRAME");
+
+			Type(
+				key_fieldLabel = "Name",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "PG1");
+
+			Button.clickSubmitButton();
+		}
+
+		task ("And given: An action to add a Commerce Product Group after updating a Commerce Product Group entry in the Commerce Product Group object") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Product Group");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				actionName = "CustomAction",
+				active = "true",
+				inputMethod = "true",
+				inputValue = "PG2",
+				objectLabel = "Commerce Product Group",
+				thenAction = "Add an Object Entry",
+				whenAction = "On After Update");
+		}
+
+		task ("When: The user updates an Commerce Product Group entry") {
+			CommerceNavigator.gotoPortlet(
+				category = "Pricing",
+				portlet = "Product Groups");
+
+			CommerceEntry.clickEntryDropdownMenuItem(
+				entryName = "PG1",
+				menuItem = "Edit");
+
+			Type(
+				key_fieldLabel = "Name",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "PG1 Edited");
+
+			Button.clickSave();
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Then: The action is triggered and the Commerce Product Group entry is added") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Product Group");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.assertObjectActions(
+				actionActive = "Yes",
+				actionLabel = "Custom Action",
+				status = "Success");
```

Comment:
We can remove these steps for two reasons. One, we want to keep this test narrow in scope, to only check if the account entry was updated. Two, we can use a generic actions test to check if the Actions UI is updated. Does that make sense?

Reply:
Hi, @andreefarias I think this step should remain, due to check if the action is triggered

Reply:
Hi @chaowang-123. It's because if we can assert that the action trigger works on the product group, we don't need to assert the status of the objects.

_The task name refers to the same assert._ 

It's a good idea to cover it manually, but in automation, we should simplify the tests in Poshi because they are high-level and expensive in CI.

I'd used it for some tests too, but Tim explained it to me.

Reply:
Hi @andreefarias Thanks for your reply, I will remove these steps.